ASP/HTML-ASP.sublime-syntax

```diff
@@ -14,9 +14,10 @@ contexts:
         - match: "%>"
           scope: punctuation.section.embedded.end.asp
           pop: true
-        - match: (').*?(?=%>)
-          scope: comment.line.apostrophe.asp
-          captures:
-            1: punctuation.definition.comment.asp
-        - include: scope:source.asp
+        - match: ''
+          push:
+            - include: scope:source.asp # don't directly push, so that the scope doesn't also contain 'source.asp ' as well as our 'source.asp.embedded.html '
+          with_prototype:
+            - match: "(?=%>)"
+              pop: true
```

Comment:
I understand what you are proposing, and I think that it is a great idea.  However, the way the syntax works at the moment is mainly just very basic "keyword" matching - it isn't very intelligent; there are no pushed contexts while inside a function body or anything, and it currently allows all sorts of invalid keyword/control-flow combinations: `set exit for nothing / 2 "hello"  end ubound vbTextCompare = if`  anyone? ;)

So I think that when the ASP syntax is improved to parse it properly, it would be the perfect time (and probably necessary, even) to implement your suggestion. Until then, I had the idea to build it so that it has at least basic passing tests for valid code or known behaviors, to act as a solid foundation for further improvements.

This is the main reason for this PR, to test that a `%>` in a string literal is treated the same way as the ASP interpreter treats it - i.e. not as part of the string but to immediately close the ASP block.